### DISCUSSION TITLE
New GUI binary name: pilot

### DISCUSSION THREAD

**Discussion by yungyuc** [solvcon/modmesh#443] [General]
For a long time we are using a non-distinguishable name "viewer" for the GUI binary for modmesh.  There is an issue to discuss the new name: #400 .

I plan to rename it to "pilot".  The sub-module `modmesh.view` will become the sub-package `modmesh.pilot`.  All GUI code should be collected under `modmesh.pilot`.  No GUI (Qt) code should be outside it.

**Comment by tigercosmos** [comment]
Why do you choose "pilot"?
